pkg/model/manifest.go

```diff
@@ -689,6 +691,11 @@ func (m *Manifest) validate() error {
 }
 
 func (b *ManifestBuild) validate() error {
+	for k, v := range *b {
```

Comment:
as part of the validation of the build section, we now check against `nil` services. I prefer this approach as it fails early the execution of the CLI. Another approach I've considered is to add a lot of checks against `nil` in the functions that iterate over the build section, but it was making the code messy and it's not as safe as this solution, because one case can always slip through and cause a panic.